src/core/server/core_app/core_app.ts

```diff
@@ -89,7 +89,8 @@ export class CoreApp {
     const resources = coreSetup.httpResources.createRegistrar(router);
 
     router.get({ path: '/', validate: false }, async (context, req, res) => {
-      const defaultRoute = await context.core.uiSettings.client.get<string>('defaultRoute');
+      const { uiSettings } = await context.core;
```

Comment:
Why did we need to split the original chained accessor `defaultRoute`? I mean, it's easier to read for sure, just curious if I'm missing something here.